json/json.go

```diff
@@ -556,6 +561,12 @@ func (enc *Encoder) SetTrustRawMessage(on bool) {
 	}
 }
 
+// SetAddExtraNewline is an extension to the standard encoding/json package which
+// allows the program to toggle the addition of a newline in Encode on or off.
+func (enc *Encoder) SetAddExtraNewline(on bool) {
```

Comment:
What do you think of this name?
```suggestion
func (enc *Encoder) SetAppendNewline(on bool) {
```